### PR TITLE
Remove dropdown from inventory detail page

### PR DESCRIPTION
**Pull request by karelhala**
VMAAS-253
RHIPLAT2-293

### CODE REVIEW

File: src/SmartComponents/Inventory/InventoryDetail.scss [hunk 1]
```diff
@@ -34,8 +34,7 @@
   .ins-entity-facts {
     div {
       span:first-child {
-        font-weight: var(--pf-global--FontWeight--bold);
-        margin-right: 15px;
+        margin-right: 0.4rem;
```
Comment (ryelo) [inline src/SmartComponents/Inventory/InventoryDetail.scss]: Can we use the `rem` mixin here? Just to stay consistent.

Comment (karelhala) [reply src/SmartComponents/Inventory/InventoryDetail.scss]: Sure thing!

**Comment by karelhala** [comment]
@ryelo can you re-review?

**Pull request merged**